src/main/kotlin/com/atlassian/performance/tools/virtualusers/LoadTest.kt

```diff
@@ -176,6 +179,7 @@ internal class LoadTest(
                 ),
                 actionMeter = ActionMeter.Builder(AppendableActionMetricOutput(segment.actionOutput))
                     .virtualUser(segment.id)
+                    .appendPostMetricHook(DrillDownHook(JavascriptW3cPerformanceTimeline(segment.driver.getDriver())))
```

Comment:
The meaningful change.

Reply:
You didn't need that for testing. Any `Scenario` (e.g. one in `jira-actions` module or in any other repo) can override the `ActionMeter` passed onto its actions. And you can create a modified copy of any `ActionMeter` via its `Builder`

Reply:
Oh, didn't think about that, I should have asked earlier x) 
So where do you think should the hook be placed finally - here or in the `com.atlassian.jira.test.performance.scenario.AllActionScenario`?

Reply:
The `ActionMeter` already has default `postMetricHooks` as an empty list. The `JavascriptW3cPerformanceTimeline` should remain opt-in, because it has a big storage and transfer overhead. So basically, it should not be put in to any JPT lib.

The end-consumers like our internal regression tests or [hardware recommendations](https://github.com/atlassian/jira-hardware-exploration) *can* opt-in if they want to. The `AbstractJiraCoreScenario` in `jira-actions` can opt-in to that, so it gets covered by the `JiraCoreScenarioIT` integration test. 
PS. now I also see that there's a dedicated `JiraCoreScenarioDevloop` in there for fast local devloop.

Reply:
In general, each JPT lib should be self-sufficient in terms of devloop. There's a high chance it already has the necessary testing harness. If not, it should be added/extended.